libs/platform/src/lib/components/wizard-generator/components/dialog-wizard-generator/dialog-wizard-generator.component.scss

```diff
@@ -0,0 +1,46 @@
+.fdp-wizard-dialog {
+
+    .fd-message-strip {
+        margin-bottom: 12px !important;
```

Comment:
For margins please check https://fundamental-styles.netlify.app/?path=/docs/layouts-margins--all-round-margin